### ISSUE TITLE
Keyboard accessibility issues

### ISSUE THREAD

**Issue by LauraOH** [SeleniumHQ/selenium-ide#560]
Improve code to allow for keyboard accessibility (particularly on dialog boxes after launch)

## Example

<a> elements without href attributes are not keyboard accessible. The dialog launch has three links that are coded this way. Add an href="#" or a tabindex=0 to make them keyboard focusable, and ensure the event handlers respond to key presses. 

Also ensure that the CSS defines a visible focus state for these and all other elements.

The "close" button on the initial launch dialog has a tabindex="-1" which removes it from the focus order.

Within the app, the "open project" icon is an event handler on a label, and cannot receive keyboard focus (the other three icons are coded as buttons).
